text/0000-gen-fn.md

```diff
@@ -147,15 +137,14 @@ need to be given arguments of type `Option`.
 
 ## Fusing
 
-Like `Generators`, `Iterator`s produced by `gen` panic when invoked again after they have returned `None` once.
-This will probably be fixed by special casing the generator impl if `Generator::Return = ()`, as we can trivially
-produce infinite values of the unit type.
+`Iterator`s produced by `gen` keep returning `None` when invoked again after they have returned `None` once.
+They do not implement `FusedIterator`, as that is not a language item, but may implement it in the future.
 
 # Reference-level explanation
 [reference-level-explanation]: #reference-level-explanation
 ## New keyword
 
-In the 2024 edition we reserve `gen` as a keyword. Previous editions will use `k#gen` to get the same features.
+In the 2024 edition we reserve `gen` as a keyword. Previous editions will use `r#gen` to get the same features.
```

Comment:
Why this change? I think `k#gen` was correct here for how to access the keyword in old editions (though we'll have to implement that). `r#gen` is what code in the new edition will use if they had something like a public API named `gen`.

Reply:
That looks to have been a search/replace error in that commit.  I've pushed a fix now.